vslib/src/SwitchStateBaseMACsec.cpp

```diff
@@ -541,15 +541,16 @@ sai_status_t SwitchStateBase::loadMACsecAttrFromMACsecSA(
 
     macsecAttr.m_an = attr->value.u8;
 
-    SAI_METADATA_GET_ATTR_BY_ID(attr, SAI_MACSEC_SA_ATTR_ENCRYPTION_ENABLE, attrCount, attrList);
-
-    macsecAttr.m_encryptionEnable = attr->value.booldata;
+    //SAI_METADATA_GET_ATTR_BY_ID(attr, SAI_MACSEC_SA_ATTR_ENCRYPTION_ENABLE, attrCount, attrList);
+    //macsecAttr.m_encryptionEnable = attr->value.booldata;
```

Comment:
Prefer not to have commented code.

Reply:
you are welcome to jump in and make a proper fix for this issue if you are knowledgeable of macsec